src/main/java/com/foundrysoftware/minecraft/plugins/PluginTemplate.java

```diff
@@ -34,4 +43,22 @@ public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command
         }
         return false;
     }
+    @Override
+    public boolean onShoot(ProjectileHitEvent e){
+       if(e.getEntity() instanceof Arrow) {
+           Arrow a = (Arrow) e.getEntity();
+
+           Location loc;
+           if (e.getHitBlock() != null) {
+               loc = e.getHitBlock().getLocation();
+           } else {
+               loc = e.getHitEntity().getLocation();
+           }
+           Bukkit.getServer().broadcastMessage(loc.toString());
+           if (a.hasCustomEffect(PotionEffectType.CONDUIT_POWER)) {
+               loc.getWorld().createExplosion(loc,10,True);
```

Comment:
you know the API better than me so I don't know that the "True" boolean means. So what we do in this case is create a constant aka private static final class level variable. The name of the variable should tell me ( the reviewer/Matt/the universe) what the value means

Reply:
True Literally means true

Reply:
I'll show you what I mean by this but I'm away from PC right now